### DISCUSSION TITLE
Artisan v2.8.2 released

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#1122] [Announcement]
Late but better than never;)

**Comment by twelvecoffee** [comment]
HELLO. I downloaded the software thru my MacBook but it wont read the eb and bt.

**Reply by bigl00z3** [reply]
You will need to provide more details (it works for me).

What kind of setup (machine, devices...).
Also upload your artisan settings file to allow others to try to understand the issue and help.
Nico

**Comment by twelvecoffee** [comment]
i am using a bideli roaster. it wont read the tempts

**Comment by twelvecoffee** [comment]
i have no problem connecting to my windows.

**Reply by bigl00z3** [reply]
There can be different issues.

- You don't have the correct drivers for your equipment on your Mac (that would not be an Artisan issue but a driver issue for your hardware). Check if your hardware needs drivers.
- You don't have the same artisan settings in between your 2 computers. You can always export your Windows artisan-settings via `Help/Save Settings...` and then import them on the Mac using `Help/Load Settings...`

And more information in your replies would be helpfull. Which model of Bideli roaster, how you connect to it, what kind of configuration...
And upload your artisan-settings here (not by email but using the Github page and rename the `artisan-settinfgs.aset` file to a `artisan-settings.txt` file before uploading) that way other users can try to better understand and help you. Just saying it doesn't work is not helpfull to understand.

Nico

**Reply by poundy** [reply]
this is NOT an issue in the release, this is 100% your settings.

**Comment by Terracotta-62** [comment]


**Reply by poundy** [reply]
Makomo has given you your answer. But please, don't pollute other people's issues with unrelated posts about your own.

**Comment by MAKOMO** [comment]
Artisan never deletes anything under C:\Program Files\Artisan. The installer of Artisan does to install a new version after it got confirmation by the user.

The about dialog shows that you are using an unstable developer version of Artisan and not the latest official release which is v2.8.2. The developer versions of Artisan are not even guaranteed to start nor to do have any working function. They are also replaced on Github on every change any developer is commiting. Go back and install v2.8.2 in the first place!
